fetch.bs

```diff
@@ -3867,6 +3892,10 @@ the request.
       wanting to do a ping without request body might do so as well, but if you do have a request
       body you definitely need this as otherwise transmit-request-body loop breaks down. -->
 
+ <li><p>If <var>timingGlobal</var> is "<code>client</code>", then set <var>timingGlobal</var> to
```

Comment:
What happens if request's client is null? Should we move this to the "if request's client is non-null" block above?

Reply:
Fixed